requirements/requirements-mandatory.txt

```diff
@@ -1,3 +1,3 @@
 pynliner
 mock
-css_inline==0.8.3
+css_inline>=0.8.3
```

Comment:
have you validated this builds in Mac M1? ref. https://github.com/draytechnologies/django-mailcss/pull/1

Reply:
yeah, it is fine right now

<img width="1273" alt="Screenshot 2024-08-09 at 11 50 31" src="https://github.com/user-attachments/assets/168afda2-3007-4bdb-ba18-765fb2be4d9b">

this image is from me installing the dependencies on my mac using python 3.12

Reply:
```suggestion
css_inline
```

may want to go back to what we had before then